USAGE_android.md

```diff
@@ -413,8 +619,215 @@ Key(s) | Key code(s) | Action
 Space, p | Space = 62, p = 44 |Toggle pause/play.
 D-pad right, n | D-pad right = 22, n = 42 | Advance to the next frame when paused.
 
-### Virtual Swapchain
 
-During replay, swapchain indices for present can be different from captured indices. Causes for this can include the swapchain image count differing between capture and replay, and `vkAcquireNextImageKHR` returning a different `pImageIndex` at replay to the one that was captured. These issues can cause unexpected rendering or even crashes.
+### Troubleshooting Replay of Applications
+
+There are several steps to take to determine why a replay of a capture may not
+be working.
+Attempt each of these and re-run after the suggested modifications to see if
+the problem resolves.
+
+#### Removing Unsupported Extensions
+
+When an application is performing a capture, there is a main difference from
+normal running, the layer is present.
+One thing the layer does is expose the `VK_EXT_tooling_info` extension.
+If the application, or OS uses this extension, it may not be present during
+replay and it will fail `vkCreateDevice` with `VK_ERROR_EXTENSION_NOT_PRESENT`.
+
+To remedy this, use the `--remove-unsupported` argument to the
+`gfxrecon.py replay` command.
+
+For example (modifying the example above):
+
+```bash
+./android/scripts/gfxrecon.py replay --remove-unsupported /sdcard/gfxrecon_capture_20221210T120558.gfxr
+```
+
+#### Virtual Swapchain
+
+During replay, swapchain indices for present can be different from captured
+indices.
+Causes for this can include the swapchain image count differing between capture
+and replay, and `vkAcquireNextImageKHR` returning a different `pImageIndex` at
+replay to the one that was captured.
+These issues can cause unexpected rendering or even crashes.
+
+Virtual Swapchain insulates higher layers in the Vulkan stack from these
+problems by creating a set of images, exactly matching the swapchain
+configuration at capture time, which it exposes for them to render into.
+Before a present, it copies the virtual image to a target swapchain image for
+display.
+Since this issue can happen in many situations, virtual swapchain is the default
+setup.
+If the user wants to bypass the feature and use the captured indices to present
+directly on the swapchain of the replay implementation, they should add the
+`--use-captured-swapchain-indices` option when invoking `gfxrecon-replay`.
+
+
+## Example of Capture and Replay
```

Comment:
Brand new section.  I wanted a step-by-step example for those who have never used a tool like this.